Osiris/GUI.cpp

```diff
@@ -1365,7 +1366,16 @@ void GUI::renderMiscWindow(bool contentOnly) noexcept
         ImGui::EndPopup();
     }
     ImGui::PopID();
-
+    std::string cvar;
```

Comment:
make a static string

Reply:
then itwould only change the first command entered

Reply:
nice men1! i voted for you !!! hope yo u win election!!

Reply:
and when it has set, reset this string value